src/libsync/syncjournaldb.cpp

```diff
@@ -436,6 +443,14 @@ bool SyncJournalDb::checkConnect()
     _insertChecksumTypeQuery.reset(new SqlQuery(_db));
     _insertChecksumTypeQuery->prepare("INSERT OR IGNORE INTO checksumtype (name) VALUES (?1)");
 
+    _getDataFingerprintQuery.reset(new SqlQuery(_db));
+    _getDataFingerprintQuery->prepare("SELECT fingerprint FROM datafingerprint");
+
+    _setDataFingerprintQuery1.reset(new SqlQuery(_db));
+    _setDataFingerprintQuery1->prepare("DELETE FROM datafingerprint;");
```

Comment:
Do you actually need this query? Why not ```INSERT OR REPLACE```?
If you need it: Why is it not named _deleteDataFingerprintQuery?